packages/xo-server/src/xo-mixins/backups.js

```diff
@@ -621,7 +623,7 @@ export default class {
       fullVdisRequired,
       disableBaseTags: true,
     })
-    $defer.onFailure(() => xapi.deleteVm(delta.vm.uuid))
+    $defer.onFailure(() => xapi.VM_destroy(delta.vm.$ref))
```

Comment:
Cannot do that.

Maybe, just after exporting, you can declare a new variable:

```js
const exportedVmRef = await xapi.call('VM.get_by_uuid', delta.vm.uuid)
```

Reply:
`this._xo.getXapiObject(delta.vm.uuid).$destroy()` would'nt work here ? Why  ?

Reply:
It should be working, but we should avoid mixing XO and XAPI object as much as possible (and there are cache issues involved as well).

IMHO, simply declaring `exportedVmRef` on top would be easier/safer.